server/Cargo.toml

```diff
@@ -106,6 +106,8 @@ path-clean = "1.0.1"
 prost = "0.13.3"
 prometheus-parse = "0.2.5"
 sha2 = "0.10.8"
+kafka = "0.10.0"
```

Comment:
Using https://github.com/fede1024/rust-rdkafka instead of kafka-rust could allow the implementation to be async